ansible/roles/dev-desktop/tasks/usermod.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Check user {{ item.username }} exists
+  ansible.builtin.getent:
+    database: passwd
+    key: "{{ item.username }}"
+  register: user_exists
+  failed_when: false
```

Comment:
I tested this on `staging` and think that this needs to be:

```suggestion
  ignore_errors: yes
```

Otherwise the playbook will continue with the `Set the user shell` task and create the missing user.

Reply:
yes, you're absolutely right